plugins/mcreator-localization/lang/texts.properties

```diff
@@ -922,6 +923,8 @@ blockly.block.entity_get_spawn_x=Get X spawn coordinate of %1 for its current wo
 blockly.block.entity_get_spawn_y=Get Y spawn coordinate of %1 for its current world %2
 blockly.block.entity_get_spawn_z=Get Z spawn coordinate of %1 for its current world %2
 blockly.block.entity_get_stepheight=Get step height of %1
+blockly.block.entity_get_use_item=Get item %1 is using
+blockly.block.entity_get_use_item_ticks=Get number of ticks %1 is using item for
```

Comment:
Better text could be "Get number of ticks %1 has been using item for" or "Get number of ticks since %1 started using an item"

Reply:
✔️ Fixed